websites/B/BeatLeader/metadata.json

```
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "205984221859151873",
+		"name": "joerkig#1337"
```

Discriminator added mistakenly by old version of `pmd`
```suggestion
		"name": "joerkig"
```